devito/finite_differences/coefficients.py

```diff
@@ -120,6 +120,7 @@ class Substitutions(object):
 
     Now define some partial d/dx FD coefficients of the Function u:
 
+    >>> import numpy as np
```

Comment:
hang on. How can [this test](https://github.com/opesci/devito/blob/master/tests/test_docstrings.py#L27) possibly have worked on CI without this import ? 

How did you add this line? did you notice anything odd?

Reply:
Sorry, I didn't understand what you mean.

Reply:
I added this line by hand. I went to use this example and noticed that import for `np` was missing.